examples/matRad_example8_protonsRobust.m

```diff
@@ -173,28 +189,33 @@
 % The goal of the fluence optimization is to find a set of bixel/spot 
 % weights which yield the best possible dose distribution according to the
 % clinical objectives and constraints underlying the radiation treatment.
+
 resultGUI = matRad_fluenceOptimization(dij,cst,pln);
 
 %% Trigger robust optimization
 % Make the objective to a composite worst case objective
-cst{ixPTV,6}{1}.robustness  = 'COWC';
-cst{ixOAR,6}{1}.robustness  = 'COWC';
-
-% Create for each VOI a second objective and use 
-% voxel wise worst case optimization VWWC.
-% cst{ixPTV,6}(2,1) = cst{ixPTV,6}(1);
-% cst{ixOAR,6}(2,1) = cst{ixOAR,6}(1);
-% cst{ixPTV,6}(2,1).robustness  = 'VWWC';
-% cst{ixOAR,6}(2,1).robustness  = 'VWWC';
 
-%%
-resultGUIrobust = matRad_fluenceOptimization(dij,cst,pln);
+ROBUST_OPT = {'none','STOCH','VWWC','VWWC_INV','COWC','OWC','PROB'};
```

Comment:
This might be a little overkill in the example. Should we just have one method here (VWWC for example) and put the whole list as a comment to extend?